src/components/LoginModal/LoginModal.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+
+import ModalWithForm from "../ModalWithForm/ModalWithForm";
+import "./LoginModal.css";
+
+const LoginModal = ({
+  handleCloseClick,
+  isOpen,
+  handleLogin,
+  handleSignupClick,
+}) => {
+  const [email, setEmail] = useState("");
+  const handleEmailChange = (e) => {
+    setEmail(e.target.value);
+  };
+
+  const [password, setPassword] = useState("");
+  const handlePasswordChange = (e) => {
+    setPassword(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    handleLogin({ email, password });
+  };
+
+  const disableSubmit = !email || !password;
+
+  return (
+    <ModalWithForm
+      title="Log In"
+      buttonText="Log In"
+      handleCloseClick={handleCloseClick}
+      isOpen={isOpen}
+      onSubmit={handleSubmit}
+      disableSubmit={disableSubmit}
+    >
+      <label htmlFor="email" className="modal__label">
+        Email{""}
+        <input
+          type="email"
+          className="modal__input"
+          id="email"
+          placeholder="Email"
+          value={email}
+          onChange={handleEmailChange}
+          required
+        />
+      </label>
+      <label htmlFor="password" className="modal__label">
+        Password{""}
+        <input
+          type="password"
+          className="modal__input"
+          id="password"
+          placeholder="Password"
+          value={password}
```

Comment:
Good job that you don't forget about `value` in the inputs if you have `onChange`